### PR TITLE
CSS autofix

### PR DESCRIPTION
**Pull request by M-OC**
- Enables autofix feature for script tags.
- Enables autofix feature for style tags
- Creates tests for both autofix features.

### CODE REVIEW

File: lint-config/stylefmt.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  'rules': {
+    'indentation': 2,
+    'length-zero-no-unit': true,
+    'number-leading-zero': 'always',
+    'number-no-trailing-zeros': true,
+    'declaration-colon-space-before': 'never',
+    'block-opening-brace-newline-after': 'always',
```
Comment (gurpreetatwal) [inline lint-config/stylefmt.js]: can we generate this file out from `stylelint.js`?

File: index.js [hunk 2]
```diff
@@ -76,6 +96,51 @@ const csslint = function(target, config, dest) {
...
+  var args = _.assign(stylefmtDefaults, config);
+
+  const modifier = function(css) {
+    var openingWhitespace = css.match(/^[\r\n\t ]+/);
+    var trailingWhitespace = css.match(/[\t ]+$/);
+    var indentation;
+    var output;
+
+    if (openingWhitespace) {
+      indentation = openingWhitespace[0].match(/^([\t ]+)/m);
```
Comment (GregChan) [inline index.js]: Can indentation be a mix of tabs and spaces? Does this mean the step in 123 reincorporates those mixed tabs and spaces?

File: test/test_index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+const {check, end} = require('./helpers/test_helpers');
```
Comment (gurpreetatwal) [inline test/test_index.js]: `'use strict';`

File: lib/util.js [hunk 4]
```diff
@@ -1,17 +1,58 @@
 'use strict';
 
+/**
+ * Returns a regex that can be used for matching html tags
+ * and their content
+ *
+ * @param {string} tagName the type of html tag to be matched
+ *
+ * @returns {regex} a regex that will produce capture groups for open/closing tags and content.
```
Comment (gurpreetatwal) [inline lib/util.js]:
```diff
- regex
+ RegExp
```

**Pull request merged**